impl/client/src/main/javascript/web/prompting/components/CompositeComponent.js

```diff
@@ -127,6 +127,9 @@ define([ 'common-ui/jquery-clean', 'cdf/components/BaseComponent', 'cdf/dashboar
       var $htmlObject = $('#' + this.htmlObject);
       $htmlObject.html(html);
 
+      a11yUtil.makeAccessibleActionButton(dom.byId('edit-parameter'));
+      a11yUtil.makeAccessibleActionButton(dom.byId('remove-parameter'));
+
```

Comment:
These components are not known by the generic `CompositePanel` component. You should do this, ideally, directly on the corresponding button component classes.

I could not find literal references to these ids in source code. Which buttons are these? Also, PIR's edit and remove buttons which show on the top-right of each parameter of the prompt panel do not have these ids.
<img width="411" alt="PIR_PromptPanel_UI" src="https://github.com/pentaho/pentaho-platform-plugin-common-ui/assets/284983/356e6c72-8992-4dc5-a444-3ebf9a7c1c57">
<img width="269" alt="PIR_PromptPanel_HTML" src="https://github.com/pentaho/pentaho-platform-plugin-common-ui/assets/284983/ef525d4c-f825-46c2-8342-3f18f26a1226">

Reply:
In the meantime, realized that these are new ids added by the related PR: https://github.com/pentaho/pentaho-platform-plugin-interactive-reporting/compare/2f4460f215498ec26509e6564ac46d7c4a0182aa..8bf14fae6193ee1d1b87ef0c0c37bbe64f7816b1#diff-102f7cf40d91144bacc4353f2844ef8ea61f5538fc04ca78742be262472de3d8R34.

The code which attaches these event handlers must move to PIR's `ParameterPanelComponent`, the class which defines these button-like elements. Possibly, override the `update()` method and do this code after calling the base implementation.

Reply:
Closing this PR as the corresponding changes are no longer needed according to the appropriate changes requested to a file belonging to another repository.